src/settings/Settings.ts

```diff
@@ -186,6 +186,12 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_FEATURE,
         default: false,
     },
+    "feature_dehydration": {
+        isFeature: true,
+        displayName: _td("Offline encrypted messaging using dehydrated devices."),
```

Comment:
Drop the full stop to match other features.

```suggestion
        displayName: _td("Offline encrypted messaging using dehydrated devices"),
```